lms/static/scripts/frontend_apps/components/LaunchErrorDialog.js

```diff
@@ -85,6 +85,22 @@ export default function LaunchErrorDialog({
           </ul>
         </ErrorModal>
       );
+
+    case 'd2l_file_not_found_in_course':
+      return (
+        <ErrorModal
+          busy={busy}
+          error={error}
+          onRetry={onRetry}
+          title="Hypothesis couldn't find the file in the course"
+        >
+          <p>
```

Comment:
D2L files is not yet released so the actual wording here is not that important.

Tracking getting the final message (once course copy is done for D2L) here https://github.com/hypothesis/lms/issues/4946